crates/typst-library/src/layout/par.rs

```diff
@@ -1256,8 +1312,10 @@ fn line<'a>(
 
         // Deal with CJK punctuation at line ends.
         let gb_style = is_gb_style(shaped.lang, shaped.region);
-        let end_cjk_punct = trimmed
-            .ends_with(['”', '’', '，', '。', '、', '：', '；', '》', '）', '』', '」']);
+        let maybe_adjust_last_glyph = trimmed
+            .ends_with(['”', '’', '，', '。', '、', '：', '；', '》', '）', '』', '」'])
```

Comment:
A bit unrelated to this PR, but maybe we want to extract this list into a constant or `is_...` function.

Reply:
This list cannot be reused in other places. Anyway I'll extract it into a constant.